data/en/cachegetmetadata.json

```diff
@@ -1,13 +1,14 @@
 {
 	"name":"cacheGetMetadata",
 	"type":"function",
-	"syntax":"cacheGetMetadata(id)",
+	"syntax":"cacheGetMetadata(id [, template] [, region])",
 	"returns":"struct",
 	"related":[],
 	"description":" Gets the metadata values for a cached object.",
 	"params": [
-		{"name":"id","description":"No Help Available","required":true,"default":"","type":"string","values":[]}
-
+		{"name":"id","description":"The ID of the cached object.","required":true,"default":"","type":"string","values":[]},
+		{"name":"template","description":"Gets metadata for template caches.","required":false,"default":"","type":"string","values":[]},
+		{"name":"region","description":"(Optional) The name of the cache region","required":false,"default":"","type":"string","values":[]}
```

Comment:
You could omit "(Optional)" in description as well because that will already be indicated by setting required to false.